README.md

```diff
@@ -44,7 +49,7 @@ We use Lerna to manage multiple Cumulus packages in the same repo. You need to i
 
 We use npm for local package management
 
-    $ npm install
+    $ npm ci
     $ npm run bootstrap
```

Comment:
We should be able to update `bootstrap` commands to use `ci` with the `--ci` flag, see https://github.com/mndewitt/lerna/blob/master/README.md#--ci